users/chains.ts

```diff
@@ -21,6 +21,16 @@ async function solanaUsers(start: number, end: number) {
     }
 }
 
+// Last 24h active users
+async function elrondUsers() {
```

Comment:
hi, thanks for the PR

 is there a way to get daily data, or for any 24h range? else we wont be able to fetch historical data

Reply:
I FW this PR to someone from MvX core team that managed their APIs for analytics, maybe they can help us with the best option.